python/paddle/fluid/tests/unittests/test_elementwise_pow_op.py

```diff
@@ -35,6 +35,26 @@ def test_check_grad_normal(self):
         self.check_grad(['X', 'Y'], 'Out')
 
 
+class TestElementwisePowOp_big_shape_1(TestElementwisePowOp):
+    def setUp(self):
+        self.op_type = "elementwise_pow"
+        self.inputs = {
+            'X': np.random.uniform(0.1, 1, [100, 100]).astype("float64"),
```

Comment:
@zhupengyang Is this `shape[100,100]` too large? And does this Op still not satisfy the `all cases use large shape` like in #22065